docs/tasks_snapclones.md

```diff
@@ -6,8 +6,72 @@ sidebar_label: Snapshots and Clones
 ------
 
 ## Snapshots using Kubectl
+For taking a snapshot, you must have a PersistentVolumeClaim for which you need to take snapshot. Once you delpoy the application , it's 
+PVC will be created by the name specified in the application deployment yaml.
+Once the application is running, you take a snapshot. To take a snapshot you must reference the PersistentVolumeClaim name in the snapshot specification that references the data for which you want to take snapshot. An example of `snapshot.yaml` file given below.
+
+```
+apiVersion: volumesnapshot.external-storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: snapshot-demo
+  namespace: default
+spec:
+  persistentVolumeClaimName: demo-vol1-claim
+```
+Here snapshot-demo is the snapshot name and demo-vol1-claim is the PVC name for which you are taking the snapshot.
+You can now take a snapshot using the command below.
+```
+kubectl apply -f snapshot.yaml
+```
+The command below will show the snapshots that has already been taken.
+```
+kubectl get volumesnapshot
+```
+For more details on snapshots please click [here](https://docs.openebs.io/docs/next/snapshots.html).
+
 
 ## Clones using Kubectl
+Once you have created a snapshot, you can restore it to a new PVC. For this you must have a separate StorageClass implemented by snapshot-provisioner. Then you can create a PVC referencing to this StorageClass for dynamically provisioning new PersistentVolume. 
+
+You can create a separate StorageClass by using the the below contents in an yaml file. For example we have created 'restore-storageclass.yaml'
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: snapshot-promoter
+provisioner: volumesnapshot.external-storage.k8s.io/snapshot-promoter
+```
+
+Once the yaml has heen created, you have to deploy the yaml using the command below.
+```
+kubectl apply -f restore-storageclass.yaml
+```
+
+Next you have to you have to create a PVC that will use the StorageClass to dynamically provision a PersistentVolume. You have to create a yaml file that will deploy a PVC using the below entries. Here for example we have created restore-pvc.yaml. The PVC that will be creaed here will be named as demo-snap-vol-claim
```

Comment:
You must now create a PVC...
below entries = following entries.
For example, a restore-pvc.yaml is created.
creaed = created